pkg/activator/net/revision_backends.go

```diff
@@ -109,7 +109,7 @@ type revisionWatcher struct {
 	logger        *zap.SugaredLogger
 
 	// podsAddressable will be set to false if we cannot
-	// probe a pod directly, but its cluster IP has beeen successfully probed.
```

Comment:
Sometimes people intentionally add extra letters, this doesn't appear to be such an instance